modules/administration/pages/maintenance-window.adoc

```diff
@@ -44,7 +50,7 @@ Preferable, you will run such a tool within a maintenance window; for more infor
 By default, several update channels are configured and enabled for the {susemgr} Server.
 New and updated packages will become available automatically.
 
-To keep {susemgr} up to date, you must have it connected directly to {scc}.
+To keep {susemgr} up to date, you must have it connected directly to {scc}.
```

Comment:
Well, you it's not a "must", as below we mention that you can also use RMT.

Reply:
changed it